src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/CaseRoleService.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseAssignedUserRole;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseAssignedUserRolesResource;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseRole;
+
+import java.util.List;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class CaseRoleService {
```

Comment:
There's probably a good argument for putting this method in the existing CaseAssignedRoleService rather than creating a new class but it's ok as-is.

Reply:
@jthmcts It says

@Cacheable self-invocation (in effect, a method within the target object calling another method of the target object). The cache annotation will be ignored at runtime

Reply:
huh interesting